src/types/platforms-types/multi-track-multiprint-contract.ts

```diff
@@ -59,9 +59,10 @@ const mapArtistProfile = ({ apiTrack, nft, contract }: { apiTrack: any, nft?: NF
   if (!contract) {
     throw new Error(`Contract missing for mapArtistProfile for nft ${nft.id}`)
   }
+
   return {
-    name: contract.platformId,
-    artistId: contract.platformId,
+    name: resolveArtistNameOverrides(nft, contract),
+    artistId: resolveArtistIdOverrides(nft, contract),
```

Comment:
comment on function naming: here we use resolveArtistNameOverrides whereas the others just use titleExtractor or xyzExtractor. should we use artistNameExtractor and artistIdExtractor for the function names here to be more consistent, or is there something different going on here that lead you to use a different naming convention?

Reply:
It is unfortunately a bit more nuanced. 

The extractors only take in an `NftFactory` to extract which strategy should be used when making/deriving another field. i.e. 
```ts
audioUrlExtractor = (contract: NftFactory): Extractor => { ... }
// where Extrator is:
type Extractor = (nft: NFT) => string;
```

The "resolvers" take both the `NftFactory` and the `NFT` to then automatically execute the strategy on a particular nft. i.e.

```ts
export const resolveArtistNameOverrides = (nft: NFT, contract: NftFactory): string => { ... }
```

Agree that the naming isn't doing a good job of indicating that. 
I think this file needs some care after it settles down and there's less churn.